### PR TITLE
Chat cmd win imp

### PR DESCRIPTION
**Pull request by DanielLiebler**

### CODE REVIEW

File: src/app/services/command.service.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import { Injectable } from '@angular/core';
+
+export interface Command {
+  cmd: string;
+  /* fullCMD is the full text, parameters are the parameters, parsed by using space as delimiter.
+     parameters[0] is therefore the command name */
+  function: (fullCMD: string, parameters: string[]) => void;
```
Comment (Saphs) [inline src/app/services/command.service.ts]: Please change to something like 'rawString', 'rawCMD' or something like that as it seemed confusing to me.

File: gsgs [hunk 2]
```diff
@@ -0,0 +1,184 @@
+[33md9447cfb521b2f9c6c6ef1200748508d8a58ca23[m implemented commandService. NOT TESTED, yet
+[33mb4ef800ea172025e7d41bcaab10d05dde786d08b[m add general structure for command-service
+[33mff946aa168b37c0b8a8b35fe1d0f4a52c9eab550[m Merge pull request #75 from Saphs/fixTileDescriptions
```
Comment (Saphs) [inline gsgs]: ?

File: src/app/components/game/interface/menu-bar/home-register/chat-command-list/chat-command-list.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (Saphs) [inline src/app/components/game/interface/menu-bar/home-register/chat-command-list/chat-command-list.component.spec.ts]: why add this

File: src/app/services/command.service.ts [hunk 4]
```diff
@@ -13,7 +13,7 @@ export interface Command {
   cmd: string;
   /* fullCMD is the full text, parameters are the parameters, parsed by using space as delimiter.
```
Comment (Saphs) [inline src/app/services/command.service.ts:14]: Please update the comment accordingly

**Review by Saphs — CHANGES_REQUESTED** [review]

File: src/app/components/game/interface/menu-bar/home-register/chat-command-list/chat-command-list.component.spec.ts (hunk 3 above)
Comment (DanielLiebler) [reply src/app/components/game/interface/menu-bar/home-register/chat-command-list/chat-command-list.component.spec.ts]: please review everything and check your comments for validity. This causes a lot of confusion in your coworkers

File: src/app/services/command.service.ts (hunk 4 above)
Comment (DanielLiebler) [reply src/app/services/command.service.ts:14]: NO!!

File: gsgs (hunk 2 above)
Comment (DanielLiebler) [reply gsgs]: !

File: src/app/services/command.service.ts (hunk 1 above)
Comment (DanielLiebler) [reply src/app/services/command.service.ts]: also outdated

File: src/app/services/command.service.ts [hunk 5]
```diff
@@ -11,7 +11,7 @@ import { ObjectLoaderService } from './object-loader.service';
 
 export interface Command {
   cmd: string;
-  /* fullCMD is the full text, parameters are the parameters, parsed by using space as delimiter.
+  /* rawCMD is the full text, parameters are the parameters, parsed by using space as delimiter.
```
Comment (Saphs) [inline src/app/services/command.service.ts:14]: Your commit smells wierd

**Review by Saphs — APPROVED** [review]

File: src/app/services/command.service.ts (hunk 5 above)
Comment (DanielLiebler) [reply src/app/services/command.service.ts:14]: no it does not

**Pull request merged**